src/syngen/ml/reporters/reporters.py

```diff
@@ -76,16 +78,16 @@ def preprocess_data(self):
             )
 
         int_columns = date_columns | int_columns
-
-        original = text_to_continuous(original, str_columns).drop(str_columns, axis=1)
-        synthetic = text_to_continuous(synthetic, str_columns).drop(str_columns, axis=1)
+        text_columns = str_columns | long_text_columns
+        original = text_to_continuous(original, text_columns).drop(text_columns, axis=1)
+        synthetic = text_to_continuous(synthetic, text_columns).drop(text_columns, axis=1)
 
         for col in [i + "_word_count" for i in str_columns]:
```

Comment:
I guess it's better to replace [i + "_word_count" for i in **str_columns**] to [i + "_word_count" for i in **text_columns**]

Reply:
Thanks) I got it